travis/test_module

```diff
@@ -0,0 +1,341 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# TODO list:
+# - add doctest, add a script parameter to run doctest.
+# - add a script parameter to select the loggin level (w/default).
+
+import argparse
+import os
+import subprocess
+import logging
+
+
+BLACK, RED, GREEN, YELLOW, BLUE, MAGENTA, CYAN, WHITE = range(8)
+
+# The background is set with 40 plus the number of the color, and the
+# foreground with 30
+
+# These are the sequences need to get colored ouput
+RESET_SEQ = "\033[0m"
+COLOR_SEQ = "\033[1;%dm"
+BOLD_SEQ = "\033[1m"
+
+
+def formatter_message(message, use_color=True):
+    if use_color:
+        message = message.replace("$RESET", RESET_SEQ).replace(
+            "$BOLD", BOLD_SEQ)
+    else:
+        message = message.replace("$RESET", "").replace("$BOLD", "")
+    return message
+
+COLORS = {
+    'WARNING': YELLOW,
+    'INFO': GREEN,
+    'DEBUG': BLUE,
+    'CRITICAL': YELLOW,
+    'ERROR': RED
+}
+
+
+class ColoredFormatter(logging.Formatter):
+    def __init__(self, msg, use_color=True):
+        logging.Formatter.__init__(self, msg)
+        self.use_color = use_color
+
+    def format(self, record):
+        levelname = record.levelname
+        if self.use_color and levelname in COLORS:
+            levelname_color = COLOR_SEQ % (
+                30 + COLORS[levelname]) + levelname + RESET_SEQ
+            record.levelname = levelname_color
+        return logging.Formatter.format(self, record)
+
+
+# Custom logger class with multiple destinations
+class ColoredLogger(logging.Logger):
+
+    FORMAT = "[$BOLD%(name)s$RESET][%(levelname)s] %(message)s"
+    COLOR_FORMAT = formatter_message(FORMAT, True)
+
+    def __init__(self, name):
+        logging.Logger.__init__(self, name, logging.DEBUG)
+
+        color_formatter = ColoredFormatter(self.COLOR_FORMAT)
+
+        console = logging.StreamHandler()
+        console.setFormatter(color_formatter)
+
+        self.addHandler(console)
+        return
+
+
+
+class CheckModuleDescription(object):
+
+    description = """
+    This script will check that all modules to ensure that:
+
+        - All modules have a description README.rst file in the main direcroty.
+        - The README.rst file has the proper rst syntax to ensure that there
+          is not error when displaying module information in the odoo instance.
+        - All the modules must have an module icon file in static/description/
+        - If the module have a doc directory must have an index.rst file
+          with valid rst syntax. source: https://www.odoo.com/apps/upload/
+    """
+
+    epilog = '\n'.join([
+        "Odoo Developer Comunity Tool",
+        "Development by Katherine Zaoral (github: @zaoral)",
+        "Coded by Katherine Zaoral <kathy@vauxoo.com>.",
+        "Source code at git:Vauxoo/addons-vauxoo."
+    ])
+
+    def __init__(self):
+        """
+        Initialization of the class.
+        @return: None
+        """
```

Comment:
The init clas is not documented at least it do something special..... but document saying "IInitialization of the class" is not correct is better save such documentation.